src/elements/content-sidebar/activity-feed/SelectableActivityCard.tsx

```diff
@@ -13,8 +13,12 @@ export type Props = {
 const SelectableActivityCard = ({ children, className, isDisabled = false, onSelect, ...rest }: Props): JSX.Element => {
     const ref = React.useRef<HTMLDivElement | null>(null);
 
-    const handleClick = (event: React.SyntheticEvent<HTMLDivElement>) => {
-        if (isDisabled) {
+    const isUserLink = (event: React.SyntheticEvent<HTMLDivElement>): boolean => {
+        return event.target instanceof HTMLElement && event.target.classList.contains('bcs-UserLink');
```

Comment:
Can we avoid breaking encapsulation via the explicit class name check? Do we need this `instanceof` check, since the target is defined in the parameter type?